### ISSUE TITLE
error in query77a

### ISSUE THREAD

**Issue by gregrahn** [gregrahn/tpcds-kit#30]
As of v2.4 query77a is missing `cr_call_center_sk` in the select list and group by making it semantically different

query77a
```sql
 cr as
 (select
        sum(cr_return_amount) as returns,
        sum(cr_net_loss) as profit_loss
 from catalog_returns,
      date_dim
 where cr_returned_date_sk = d_date_sk
       and d_date between cast('[SALES_DATE]' as date)
                  and (cast('[SALES_DATE]' as date) +  30 )
 ),
```

query77
```sql
cr as
 (select cr_call_center_sk,
         sum(cr_return_amount) as returns,
         sum(cr_net_loss) as profit_loss
 from catalog_returns,
      date_dim
 where cr_returned_date_sk = d_date_sk
       and d_date between cast('[SALES_DATE]' as date)
                  and (cast('[SALES_DATE]' as date) +  30 days)
 group by cr_call_center_sk
 ), 
```

**Comment by gregrahn** [comment]
Fixed in this repo via 17d24b576906ca91a1e653b05685fb7e6488e4aa.

**Comment by gregrahn** [comment]
Present in v2.4.0

**Comment by gregrahn** [comment]
Present in v2.6.0

**Comment by gregrahn** [comment]
Fixed in v2.9.0
